ratis-server-api/src/main/java/org/apache/ratis/server/RaftServerConfigKeys.java

```diff
@@ -116,6 +116,21 @@ static void setStagingCatchupGap(RaftProperties properties, int stagingCatchupGa
     setInt(properties::setInt, STAGING_CATCHUP_GAP_KEY, stagingCatchupGap);
   }
 
+  String BOOTSTRAP_TIMEOUT_KEY = PREFIX + ".bootstrap.timeout";
```

Comment:
Since we already have `staging.catchup.gap`, how about calling it `staging.timeout`?  It was unfortunate that we already used `staging` instead of `bootstrap`.

Reply:
No problem, let's use `staging.timeout` for consistency.